rpm/fedora-27/Dockerfile.x86_64

```diff
@@ -0,0 +1,17 @@
+FROM fedora:27
+RUN dnf -y upgrade
+RUN dnf install -y @development-tools fedora-packager
```

Comment:
Should we follow best-practice here, and combine `dnf -y upgrade` with the `dnf install` lines? (I guess we don't use caching, so probably doesn't make a big difference)

Reply:
Yeah these aren't cached when we build packages since we build on freshly spawned slaves in the Jenkins setup, WDYT @andrewhsu?

Reply:
It's just a nit, definitely no blocker for this PR